src/components/Select/Select.tsx

```diff
@@ -122,7 +123,7 @@ export const Select = <ValueType, >(
       optionRender={optionRender}
       options={options}
       placeholder={placeholder}
-      popupMatchSelectWidth={false}
+      popupMatchSelectWidth={popupMatchSelectWidth}
```

Comment:
is this a boolean prop? it's name should start with a verb such as `shouldPopupMatchSelectWidth` or `mustMatchPopupSelectWidth`

Reply:
https://github.com/mia-platform/design-system/pull/938#discussion_r2077034626